gh_pages/_source/demo1/Inspect-the-package.md

```diff
@@ -10,6 +10,7 @@ catkin init
 ```
 
 ## Download debian dependencies
+>Clone the ROS-I repo [universal_robot](https://github.com/ros-industrial/universal_robot) into the src directory of `perception_driven_ws`. Switch to the `melodic-devel` branch.
```

Comment:
`kinetic-devel` is actually the default "master" branch of the UR repository. I would amend this to say:

>Clone the [universal_robot repository](https://github.com/ros-industrial/universal_robot) into the `src` directory of `perception_driven_ws`